patroni/postgresql.py

```diff
@@ -981,7 +1148,7 @@ def clone(self, clone_member):
 
     def bootstrap(self, config):
         """ Initialize a new node from scratch and start it. """
-        if self._initialize(config) and self.start():
+        if self._initialize(config) and self.start() and self.wait_for_startup():
```

Comment:
Here is a small conflict: https://github.com/zalando/patroni/commit/48a6af6994f2bbb8a13e58db6179a514a3937429#diff-d8c1582337c8426ef750420dc6b76949R1008